### DISCUSSION TITLE
How can I setup a options page?

### DISCUSSION THREAD

**Discussion by leandroribeir0** [crxjs/chrome-extension-tools#507] [Q&A] [answered]
I need a options page, but I don't know how configure to achieve that.

**Comment by jacksteamdev** [comment, marked as answer]
You can add it in the manifest under `options_page`. 

See Google documentation for details: [Give users options - Chrome Developers](https://developer.chrome.com/docs/extensions/mv3/options/)

**Comment by leandroribeir0** [comment]
I just need to add in the manifest? Simple as that?
